conf/ocsci/aws_i3_local_storage.yaml

```diff
@@ -0,0 +1,8 @@
+---
+DEPLOYMENT:
+  local_storage: true
+
+ENV_DATA:
+  platform: 'aws'
+  worker_instance_type: 'i3en.2xlarge'
+  device_size: 2300
```

Comment:
Can you please comment from which place this value came?

Reply:
Done